CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.11](https://github.com/brave/brave-ios/releases/tag/v1.11)
+ 
+ - Added quick action for New Private Tab from home screen. ([#1258](https://github.com/brave/brave-ios/issues/1258))  
+ - Improved video orientation on Youtube. ([#1189](https://github.com/brave/brave-ios/issues/1189))
+ - Fixed passcode not being reset when resuming from the background. ([#1203](https://github.com/brave/brave-ios/issues/1203))
```

Comment:
nit: I think `when resuming from background` sounds better compared to `when resuming from the background`

Reply:
changed